feature-configs/demo/ptp/ptpconfig-slave.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
```

Comment:
Now that the slave profile is in base, can we remove this (or maybe just have this as a little patch?)